### PR TITLE
Include component generator into the App Generator

### PR DESCRIPTION
**Pull request by animesh10**
generator

### CODE REVIEW

File: packages/generator-electrode/component/templates/test/client/eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+extends:
+  - "../../node_modules/electrode-archetype-react-component/config/eslint/.eslintrc-react-test"
```
Comment (jchip) [inline packages/generator-electrode/component/templates/test/client/eslintrc]: remove `../../node_modules/`

File: packages/generator-electrode/README.md [hunk 2]
```diff
@@ -42,6 +42,47 @@ Some common ones:
   - `gulp server-prod` - start server in production mode
   - `gulp check` - run unit tests with coverage
 
+## Generating a React Component
+  Install the generator if you haven't already:
+
+  ```bash
+  npm install -g generator-electrode
```
Comment (didi0613) [inline packages/generator-electrode/README.md]: Does this needs to be install globally?

Comment (jchip) [inline packages/generator-electrode/README.md]: That's the easiest way, since you don't have your own project yet when you need to use the generator.

**Comment by didi0613** [comment]
LGTM 👍

**Pull request merged**